tsconfig.json

```diff
@@ -12,91 +12,107 @@
     ],
     "target": "es6",
     "module": "commonjs",
-    "baseUrl": "src/main",
+    "baseUrl": "src" ,
     "paths": {
-      "app/*": [
-        "app/*"
-      ],
+
       "payment-hub-client/*": [
-        "app/pay/*"
+        "main/app/pay/*"
       ],
       "client/*": [
-        "app/client/*"
+        "main/app/client/*"
       ],
       "postcode-lookup/*": [
-        "app/postcode-lookup/*"
+        "main/app/postcode-lookup/*"
       ],
       "drafts/*": [
-        "app/drafts/*"
+        "main/app/drafts/*"
       ],
       "claims/*": [
-        "app/claims/*"
+        "main/app/claims/*"
+      ],
+      "common/*": [
+        "main/app/common/*"
+      ],
+      "documents/*": [
+        "main/app/documents/*"
       ],
       "fees/*": [
-        "app/fees/*"
+        "main/app/fees/*"
       ],
       "forms/*": [
-        "app/forms/*"
+        "main/app/forms/*"
       ],
       "guards/*": [
-        "app/guards/*"
+        "main/app/guards/*"
       ],
       "health/*": [
-        "app/health/*"
+        "main/app/health/*"
       ],
       "idam/*": [
-        "app/idam/*"
+        "main/app/idam/*"
       ],
       "logging/*": [
-        "app/logging/*"
+        "main/app/logging/*"
       ],
       "models/*": [
-        "app/models/*"
+        "main/app/models/*"
+      ],
+      "pdf/*": [
+        "main/app/pdf/*"
       ],
       "utils/*": [
-        "app/utils/*"
+        "main/app/utils/*"
+      ],
+      "paths": [
+        "main/app/paths"
+      ],
+      "errors" :[
+        "main/errors"
       ],
       "modules/*": [
-        "modules/*"
+        "main/modules/*"
       ],
       "routes/*": [
-        "routes/*"
+        "main/routes/*"
       ],
-      "common/*": [
-        "common/*"
+      "shared/*": [
+        "main/common/*"
```

Comment:
to avoid confusion, should we rename the dir to ` "main/shared/*"`?

Reply:
Shared is the short term solution. We plan to merge these in the future.